server/routes/barcode/image/GenerateBarcodeImageView.ts

```diff
@@ -1,6 +1,7 @@
 type BarcodeImage = {
   barcodeImageUrl: string
   barcodeImageName: string
+  recipientName: string
```

Comment:
The recipient name needs to be added to the barcode image data in the view, so that the nunjucks template can render it as part of the Copy and Download buttons